packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -103,6 +103,7 @@ export class StripeHelper {
       : undefined;
 
     this.stripe = new Stripe(config.subscriptions.stripeApiKey, {
+      // @ts-ignore
```

Comment:
What is being ignored?

Reply:
Stripe has updated their version, according to their docs, this is the work around until we update to the latest API version.